pyspecProcScripts/load_data.py

```diff
@@ -629,6 +629,9 @@ def proc_spincore_ODNP_v4(s, fl=None):
         s.ft("t2")
     return s
 
+def proc_spincore_generalproc_v1(s, fl=None):
+    for j in [k for k in s.dimlabels if k.startswith('ph')]:
+        s.ft(j)
```

Comment:
typically, I think we also want to:

- FT along `t2`
- `.squeeze()`

Reply:
but discuss w/ AG

Reply:
for the rest of the preprocessing functions we save the squeeze for after - e.g. we don't include here